std/meta.d

```diff
@@ -61,6 +61,67 @@ unittest
 }
 
 /**
+ * Not everything can be directly aliased. An alias cannot be declared
```

Comment:
How about: "Enables alias declarations for symbols where such are not normally permitted." – followed by a blank line and then what you've already written?